### ISSUE TITLE
"Biggest available win" should focus on WCAG corrections

### ISSUE THREAD

**Issue by maggiewachs** [mgifford/vital-core#262]
Example page: https://mgifford.github.io/vital-core/reports/www.cdc.gov/2026-W37/index.html

The "biggest available win" on the report linked above (and several others) flags an axe rule that is qualified as a best practice, not a WCAG violation. Should call-outs like this be limited to WCAG?
